Jordan nash fix intro md

LayersOfAbstraction opened the pull request:
Fixes #24541 and #26505

Hey again @tdykstra, I know I sound like a robot here but to avoid confusion I think It's best to repeat everything. in the PR #27440 (which this is a parent to this PR) you mentioned the following:

> * Perhaps mention a few of the other changes required by .NET 6 or 7.

> * Briefly explain that NRT (nullable reference types) is enabled by default in .NET 6 or 7, mention a couple of the problems that causes, and link to https://learn.microsoft.com/ef/core/miscellaneous/nullable-reference-types for more information. This would be a good place to bring up ModelOnly vs. All and explain how to disable NRT for people who don't want to deal with it.
Perhaps mention a few of the other changes required by .NET 6 or 7.
<!--

I described all that in Perquisites as briefly as I could. Hope it helps.
# Instructions

When creating a new PR, please reference the issue number if there is one:

Fixes #Issue_Number

The "Fixes #nnn" syntax in the PR description allows GitHub to automatically close the issue when this PR is merged.

NOTE: This is a comment; please type your descriptions above or below it.
-->

tdykstra reviewed (commented):
Thanks for submitting this PR. I reworked your text into more of a warning to expect problems ahead if .NET 6 or later is used for the tutorial.

Please revert the changes to support .NET 6, as updating the tutorial for .NET 6 would take much more work than updating the references in this file and we don't yet have resources to devote to that project.

LayersOfAbstraction commented on aspnetcore/data/ef-mvc/intro.md:
```diff
@@ -26,7 +26,13 @@ The sample app is a web site for a fictional Contoso University. It includes fun
-[!INCLUDE[VS prereqs](~/includes/net-core-prereqs-vs-5.0.md)]
+[!INCLUDE[VS prereqs](~/includes/net-prereqs-vs-6.0.md)]
+
+* Please advise that the .NET 6 & 7 NRTs [(nullable reference types)](https://learn.microsoft.com/ef/core/miscellaneous/nullable-reference-types ) is enabled by default. Problems can happen where we're not creating navigation property elements which would be required for object creation. We will touch on how to fix this soon and will patch this in future documentation.
+
+* .NET 6 by default has the `asp-validation-summary` helper tag set to `ModelOnly` which prevents notifying us of problems like the one we described with NRTs. We will show how to correct that in this tutorial but you can ignore it by disabling NRTs which we will touch on, in this page. 
+
+* You want to try and use the minimal hosting model generated in .NET 6 though it is not mandatory, it unifies Program.cs and.
```
What you mean by revert changes? You mean I commit your suggestion? I’m confused. Am I supposed to hold off?

tdykstra replied:
>You mean I commit your suggestion?

Yes, and those additions to the Prerequisites section would be the only changes we make at this time in this PR, so the changes outside of the Prerequisites section would need to be undone. The problem is we can't just update one tutorial -- they all have to be updated at the same time, as each successive tutorial in the series builds on what was done in the preceding ones. That's why all we're prepared to accept now is an update to the intro page warning that the tutorial series is not yet updated for .NET 6 or 7.

As [suggested by Rick-Anderson](https://github.com/dotnet/AspNetCore.Docs/pull/27440#issuecomment-1306648126), at some later date we will be ready to work on updating this tutorial series, and we'll be able to take more contributions then.

LayersOfAbstraction commented:
Ok, it looks like I have removed everything outside the Prequists section and reverted it back as you asked. I forgot I messed with that ## Create web ## section before. That said I am unsure if your 2 suggested changes get pushed. Do I need to unresolve and then commit them again? Or do they get auto-committed anyway regardless of the changes I made? Also is there anything else I missed?

tdykstra commented on aspnetcore/data/ef-mvc/intro.md:
```diff
@@ -383,16 +388,16 @@ Advance to the next tutorial to learn how to perform basic CRUD (create, read, u
 
-* [.NET Core SDK 2.2](https://dotnet.microsoft.com/download)
-* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=learn.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) with the following workloads:
+* [.NET Core SDK 6.0](https://dotnet.microsoft.com/download)
+* [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=learn.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2022) with the following workloads:
```
This is in the moniker section for earlier than 3.1, so it can't be updated to .NET 6 / VS2022.

```suggestion
* [.NET Core SDK 2.2](https://dotnet.microsoft.com/download)
* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=learn.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) with the following workloads:
```

tdykstra commented on aspnetcore/data/ef-mvc/intro.md:
```diff
@@ -383,7 +388,7 @@ Advance to the next tutorial to learn how to perform basic CRUD (create, read, u
 [!INCLUDE [RP better than MVC](~/includes/RP-EF/rp-over-mvc.md)]
 
-The Contoso University sample web application demonstrates how to create ASP.NET Core 2.2 MVC web applications using Entity Framework (EF) Core 2.2 and Visual Studio 2017 or 2019.
+The Contoso University sample web application demonstrates how to create ASP.NET Core 6.0 MVC web applications using Entity Framework (EF) Core 6.0.9 and Visual Studio 2022 or 2022.
```
This is in the moniker section for earlier than 3.1, so it can't be updated to .NET 6 / VS2022.

```suggestion
The Contoso University sample web application demonstrates how to create ASP.NET Core 2.2 MVC web applications using Entity Framework (EF) Core 2.2 and Visual Studio 2019.
```

tdykstra reviewed (approved):
@LayersOfAbstraction Thank you for working with us to get this updated, and we look forward to accepting more of your changes when the whole series is updated.

tdykstra commented:
>That said I am unsure if your 2 suggested changes get pushed. Do I need to unresolve and then commit them again? Or do they get auto-committed anyway regardless of the changes I made? Also is there anything else I missed?

I don't think there's anything you missed, aside from a few changes in the 2.2 Moniker section, which I reverted.
